researcher_ids/RI_Claims_V1.md

```diff
@@ -237,38 +245,41 @@ for more details).
 
 **Diagram 2: basic flow of data from Passport Visa Assertion Source to Passport Clearinghouse.**
 
-In Diagram 2, the general flow of Passport-related data from [Passport
-Visa Assertion Source](#passport-visa-assertion-source) organization to [Passport
-Clearinghouse](#passport-clearinghouse) service is shown at a high level. The
-colors used in Diagram 2 correspond to the colors of the data from Diagram 1 to
-give a sense of which services contributed the data. However, various elements
-within the [Passport Visa](#passport-visa) can be collected into standard form
-by either the [Passport Visa Assertion
+In Diagram 2, the general flow of Passport-related data from a [Passport
+Visa Assertion Source](#passport-visa-assertion-source) organization to a
+[Passport Clearinghouse](#passport-clearinghouse) service is shown at a high
+level. The colors used in Diagram 2 correspond to the colors of the data from
+[Diagram 1](#diagram-1) to give a sense of which services contributed the data.
+However, various elements within the [Passport Visa](#passport-visa) can be
+collected into standard form by either the [Passport Visa Assertion
 Repository](#passport-visa-assertion-repository) service or the
 [Passport Visa Issuer](#passport-visa-issuer) service depending on the
 protocols and procedures employed between these components.
 
-Implementations introduce clients, services, and procedures -- not shown in
-Diagram 2 -- to provide the mechanisms to move the data between the Passport
-Visa Assertion Source and the [Passport Broker](#passport-broker). This MAY be
-proprietary and is beyond the scope of this specification. The flow
-between these components (represented by black arrows) MAY not be direct or
-conversely services shown as being separate MAY be combined into one service.
-For example, some implementations MAY deploy one service that handles the
-responsiblities of both the Passport Visa Issuer and the Passport Broker.
+Implementations may introduce clients, additional services, and protocols --
+not detailed in Diagram 2 -- to provide the mechanisms to move the data between
+the Passport Visa Assertion Source and the [Passport Broker](#passport-broker).
+These mechanisms are unspecified by the scope of this specification except that
+they MUST adhere to security and privacy best practices, such as those outlined
+in the GA4GH AAI Specification, in their handling of user sessions, Passport
```

Comment:
I don't think AAI spec defines _a user session_ between the broker and assertion source. At least in ELIXIR/EGA we have none.

Reply:
Changed "user sessions" to "protocols", thanks for catching this.